config/routes.rb

```diff
@@ -11,7 +11,9 @@
 
   scope '/coordinator' do
     get '/deliver', to: 'coordinator#deliver', as: :coordinator_deliver
+    post '/deliver', to: 'coordinator#confirm', as: :coordinator_confirm
     get '/schedule', to: 'coordinator#schedule', as: :coordinator_schedule
+    post '/schedule', to: 'coordinator#match', as: :coordinator_match
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.